### DISCUSSION TITLE
How do I change the padding of a button and all the controls in DaisyUI

### DISCUSSION THREAD

**Discussion by AJ1000** [saadeghi/daisyui#2934] [Q&A] [answered]
I am having a great time with DaisyUI, but I cannot seem to get the padding of a button (and all the other controls) to be py-1.5 px-4 
 
This is the padding of the buttons I am currently using on my site, and so all the DaisyUI buttons, for example the dropdown button are bigger than the buttons I have created using tailwind itself (e.g. I made buttons that scale out, have a banner inside them etc). 

I want all the buttons to be the same size, but have not had any luck changing the DaisyUI to be py-1.5 px-4 Can anyone provide an example of getting this to work? 

Thanks

**Comment by saadeghi** [comment, marked as answer]
You can set the horizontal padding for buttons but buttons don't have vertical padding at all. They have height and min height (which you can overridde using utility classes)

Let me know if you have any questions

**Comment by AJ1000** [comment]
Thanks for the answer I will continue to experiment trying to get this to work. I am using py-1.5 px-4 in my tailwind buttons without problem. I will experient with height and minimum height when I get free time. This is what I currently have, and the vertical padding is where my issue is I think, The button with Daisy uses DaisyUI, the other button uses Tailwind directly.

![differnce](https://github.com/saadeghi/daisyui/assets/4085891/a7ed64e6-ff41-44d5-828f-6e3b5b33ddc2)

If you have any suggestions, they would be appreciated.
